network/codec/cbor/decode.go

```diff
@@ -16,74 +18,74 @@ func switchenv2v(code uint8) (interface{}, error) {
 	switch code {
 
 	// consensus
-	case CodeBlockProposal:
+	case codec.CodeBlockProposal:
```

Comment:
maybe you could take this a step further and define a new type like
```
type MessageType uint8
```

which has `String()` and `Interface()` methods you can use to get the `what` and `interface{}` values. Then all this mapping code could be defined once and we'd have a more consistent interface to work with instead of `switchenv2what`

Reply:
https://github.com/onflow/flow-go/pull/2491/commits/55201f6ad6642613e551228ba154816baaec6790